.github/workflows/gradle.yml

```diff
@@ -68,6 +68,10 @@ jobs:
         run: echo Building a '${{ github.event_name }}' for target '${{ github.ref }}'.
       - name: Checkout
         uses: actions/checkout@v2
+      - uses: actions/setup-java@v2
+        with:
+          distribution: 'adopt'
```

Comment:
From the "setup-java" documentation:
> NOTE: Adopt OpenJDK got moved to Eclipse Temurin and won't be updated anymore. It is highly recommended to migrate workflows from adopt to temurin to keep receiving software and security updates. See more details in the Good-bye AdoptOpenJDK post.

So perhaps we should use 'termurin'?

Reply:
Good catch, switched to `Temurin`.